packages/sheets-ui/src/commands/commands/__tests__/add-worksheet-merge.command.spec.ts

```diff
@@ -281,4 +331,186 @@ describe('Test add worksheet merge commands', () => {
+                        rangeType: RANGE_TYPE.COLUMN,
+                    },
+                    primary: null,
+                    style: null,
+                },
+            ]);
+            expect(await commandService.executeCommand(InsertColAfterCommand.id)).toBeTruthy();
+            expect(worksheet.getMergeData().length).toBe(2);
+            expect(worksheet.getMergeData()[1]).toStrictEqual({ startRow: 10, startColumn: 11, endColumn: 16, endRow: 15, rangeType: 0 });
+            selectionManager.clear();
+            selectionManager.add([
+                {
+                    range: {
+                        startRow: 0,
+                        startColumn: 12,
+                        endColumn: 13,
+                        endRow: worksheet.getRowCount() - 1,
+                        rangeType: RANGE_TYPE.COLUMN,
+                    },
+                    primary: null,
+                    style: null,
+                },
+            ]);
+            expect(await commandService.executeCommand(InsertColAfterCommand.id)).toBeTruthy();
+            expect(worksheet.getMergeData().length).toBe(2);
+            expect(worksheet.getMergeData()[1]).toStrictEqual({ startRow: 10, startColumn: 11, endColumn: 18, endRow: 15, rangeType: 0 });
+
+            selectionManager.clear();
+            selectionManager.add([
+                {
+                    range: {
+                        startRow: 0,
+                        startColumn: 7,
+                        endColumn: 7,
+                        endRow: worksheet.getRowCount() - 1,
+                        rangeType: RANGE_TYPE.COLUMN,
+                    },
+                    primary: null,
+                    style: null,
+                },
+            ]);
+            expect(await commandService.executeCommand(RemoveColCommand.id)).toBeTruthy();
+            expect(worksheet.getMergeData().length).toBe(2);
+            expect(worksheet.getMergeData()[1]).toStrictEqual({ startRow: 10, startColumn: 10, endColumn: 17, endRow: 15, rangeType: 0 });
+            selectionManager.clear();
+            selectionManager.add([
+                {
+                    range: {
+                        startRow: 0,
+                        startColumn: 12,
+                        endColumn: 13,
+                        endRow: worksheet.getRowCount() - 1,
+                        rangeType: RANGE_TYPE.COLUMN,
+                    },
+                    primary: null,
+                    style: null,
+                },
+            ]);
+            expect(await commandService.executeCommand(RemoveColCommand.id)).toBeTruthy();
+            expect(worksheet.getMergeData().length).toBe(2);
+            expect(worksheet.getMergeData()[1]).toStrictEqual({ startRow: 10, startColumn: 10, endColumn: 15, endRow: 15, rangeType: 0 });
+        });
+    });
+
+    describe('with move col', () => {
+        it('', async () => {
```

Comment:
Please give explanative description in test cases' titles. https://github.com/dream-num/univer-pro/blob/dev/docs/code-style-guide.md#%E8%AF%B4%E6%98%8E%E6%84%8F%E5%9B%BE